service/service.go

```diff
@@ -159,6 +161,20 @@ func NewService(ctx context.Context) (*service, error) {
 		}
 	}
 
+	go func() {
+		timer := time.NewTicker(1 * time.Hour)
+		defer timer.Stop()
+		for {
+			select {
+			case <-timer.C:
+				svc.removeExcessEvents()
+			case <-ctx.Done():
+				svc.Shutdown()
```

Comment:
This shouldn't be here, I will do a PR to remove it

Reply:
Ah true, sorry I missed it too!